config/software/bzip2.rb

```diff
@@ -40,7 +40,7 @@
   args = "PREFIX='#{install_dir}/embedded' VERSION='#{version}'"
 
   patch source: 'makefile_take_env_vars.patch', env: env
-  patch source: 'soname_install_dir.patch', env: env if mac_os_x_mavericks?
+  patch source: 'soname_install_dir.patch', env: env if mac_os_x?
```

Comment:
https://github.com/chef/omnibus-software/pull/251

It looks like the issue it was addressing was introduced in mavericks.  However, it isn't clear if the author was just being conservative or if this patch will cause problems pre-mavericks.